cypress/e2e/BasicTest.ts

```diff
@@ -1,13 +1,33 @@
 describe("Basic Elements", () => {
   const bridgeName = uniqueName("myBridge");
-  const restUrl = `${Cypress.env("REST_URL")}${Cypress.env(
-    "REST_PATH"
-  )}/bridges`;
+  const restBaseUrl = Cypress.env("SANDBOX_DEV_REST_URL");
+  const restPath = Cypress.env("SANDBOX_DEV_REST_PATH");
+  const restUrl = `${restBaseUrl}${restPath}/bridges`;
   const token = Cypress.env("OB_TOKEN");
+  const user: string = Cypress.env("USER");
+  const psw: string = Cypress.env("PASSWORD");
   const authorization = `Bearer ${token}`;
   let bridgeId: String;
 
   before(() => {
+    expect(restBaseUrl, "REST base url was set").to.be.a("string").and.not.be
+      .empty;
+    expect(restPath, "REST path was set").to.be.a("string").and.not.be.empty;
+    if (typeof token !== "string" || !token) {
+      throw new Error("Missing token value, set using CYPRESS_OB_TOKEN=...");
+    } else {
+      cy.log("Token is set correctly");
+    }
+    if (typeof user !== "string" || !user) {
+      throw new Error("Missing user value, set using CYPRESS_USER=...");
+    } else {
+      cy.log("User name is set correctly");
+    }
+    if (typeof psw !== "string" || !psw) {
+      throw new Error("Missing password value, set using CYPRESS_PASSWORD=...");
+    } else {
+      cy.log("Password is set correctly");
+    }
```

Comment:
@jomarko I have added assertation if the envs are loaded correctly and put some logs and error message for each usecase.